src/api/handlers/channel.ts

```diff
@@ -301,6 +302,38 @@ export async function createInvite(
   return RequestManager.post(endpoints.CHANNEL_INVITES(channelID), options);
 }
 
+/** Returns an invite for the given code. */
+export function getInvite(inviteCode: string) {
+  //TODO(itohatweb): types: Better return type
+  return RequestManager.get(endpoints.INVITE(inviteCode));
```

Comment:
We should copy-paste the interface from the types pr to use it here. This should not be merged knowing its missing pieces pending on a v11 pr thats a breaking change.